config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get '/guests/:id' => 'guests#show'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  post '/guest_rooms/:id' => 'guest_rooms#create'
```

Comment:
The verb, controller, and action for this route are correct, the path is unfortunately not correct when we think about parent/child routes. Your path should actually be: `"/guest/:id/rooms"`.

Path for create actions should not end in an ID.